### ISSUE TITLE
In the pipeline yarn sometimes throws a error

### ISSUE THREAD

**Issue by jameswlane** [jameswlane/status-board#60]
Problem description:
When building sometimes `eval yarn` in the Travis install fails.
```
[0Ktravis_time:start:26e0e7b8
[0K$ yarn
yarn install v1.3.2
warning npm-shrinkwrap.json found. This will not be updated or respected. See https://yarnpkg.com/en/docs/migrating-from-npm for more information.
[1/4] Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/pg: ETIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/home/travis/build/jameswlane/status-board/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

[31;1mThe command "eval yarn " failed. Retrying, 2 of 3.[0m

yarn install v1.3.2
warning npm-shrinkwrap.json found. This will not be updated or respected. See https://yarnpkg.com/en/docs/migrating-from-npm for more information.
[1/4] Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/pg: ETIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/home/travis/build/jameswlane/status-board/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

[31;1mThe command "eval yarn " failed. Retrying, 3 of 3.[0m

yarn install v1.3.2
warning npm-shrinkwrap.json found. This will not be updated or respected. See https://yarnpkg.com/en/docs/migrating-from-npm for more information.
[1/4] Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/pg: ETIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/home/travis/build/jameswlane/status-board/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

[31;1mThe command "eval yarn " failed 3 times.[0m


travis_time:end:26e0e7b8:start=1515192223965500196,finish=1515192318966681023,duration=95001180827
[0K
[31;1mThe command "yarn" failed and exited with 1 during .[0m

Your build has been stopped.
```

**Comment by SwamiKalagiri** [comment]
This issue may be related to yarn 1.3.2 [issue#4877](https://github.com/yarnpkg/yarn/issues/4877) and it look like they fixed in [PR#4903](https://github.com/yarnpkg/yarn/pull/4903) which not released yet.

**Comment by jameswlane** [comment]
This is no longer a issue since moving to CircleCi
